### PR TITLE
Add section on graphics to troubleshooting

### PR DESCRIPTION
**Pull request by Wumpf**
### Checklist
* [x] I have read and agree to [Contributor Guide](../CONTRIBUTING.md) and the [Code of Conduct](../CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by emilk — APPROVED** [review]
Awesome

**Review by emilk — COMMENTED** [review]
I think we should also mention using `WINIT_UNIX_BACKEND=x11` as something worth trying on Linux

**Comment by teh-cmc** [comment]
It would probably make sense to mention ICD files too.

For a lot of Linux users, forcing their discrete nvidia to move out of the way will fix most problems...

**Comment by Wumpf** [comment]
added note on WINIT_UNIX_BACKEND.
Puzzling something together out of Clement's comment for icd files :)

**Comment by Wumpf** [comment]
put that in, also in the Linux section. No need to scare away user of other OSes by putting it under graphics I figure.
@teh-cmc can you glimpse over it? thx

**Comment by teh-cmc** [comment]
Yep, I'll expand a bit, gimme a minute.

**Comment by teh-cmc** [comment]
Man explaining this stuff succinctly is not funny... I think it's good enough now, please check :grimacing:

File: docs/getting-started/troubleshooting.md [hunk 1]
```diff
@@ -20,3 +20,27 @@ On Fedora Rawhide you need to run:
...
+* `WGPU_BACKEND`: Overwrites the graphics backend used, must be one of `vulkan`, `metal`, `dx12`, `dx11`, or `gl`.
+    Naturally, support depends on your OS. Default is `vulkan` everywhere except on Mac where we use `metal`.
+* `WGPU_POWER_PREF`: Overwrites the power setting used for choosing a graphics adapter, must be `high` or `low`. (Default is `high`)
+
+We recommend setting these only if you're asked to try them or know what you're doing,
+since we don't support all of these settings equally well.
+
+### Multiple GPUs
+
+When using Wgpu's Vukan backend (the default on Windows & Linux) on a computer that has both integrated and dedicated GPUs, a lot of issues can arise from Vulkan either picking the "wrong" GPU at runtime, or even simply from the fact that this choice conflicts with other driver picking technologies (e.g. NVIDIA Optimus).
```
Comment (Wumpf) [inline docs/getting-started/troubleshooting.md:42]: technically it's not just vulkan, right?

File: docs/getting-started/troubleshooting.md [hunk 2]
```diff
@@ -20,3 +20,27 @@ On Fedora Rawhide you need to run:
...
+
+In both cases, forcing Vulkan to pick either the integrated or discrete GPU (try both!) using the [`VK_ICD_FILENAMES`](https://vulkan.lunarg.com/doc/view/1.3.204.1/mac/LoaderDriverInterface.html#user-content-driver-discovery) environment variable might help with crashes, artifacts and bad performance. E.g.:
+- Force the integrated GPU: `export VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/intel.json`.
+- Force the discrete GPU: `export VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/nvidia.json`.
```
Comment (Wumpf) [inline docs/getting-started/troubleshooting.md:46]: doesn't indicate this works only on linux

**Review by Wumpf — COMMENTED** [review]
good improvement. let me add anote that the fix is just for linux

**Pull request merged**